SU2_CFD/src/solver_direct_mean.cpp

```diff
@@ -799,22 +796,22 @@ CEulerSolver::CEulerSolver(CGeometry *geometry, CConfig *config, unsigned short
 
   /*--- Initialize the solution to the far-field state everywhere. ---*/
 
-  for (iPoint = 0; iPoint < nPoint; iPoint++)
-    node[iPoint] = new CEulerVariable(Density_Inf, Velocity_Inf, Energy_Inf, nDim, nVar, config);
+  node = new CEulerVariable(Density_Inf, Velocity_Inf, Energy_Inf, nPoint, nDim, nVar, config);
+  snode = static_cast<CEulerVariable*>(node);
```

Comment:
This "special" node is just a cast of the normal node, since this is done in only one place I think there is little risk of initializing it incorrectly.